fix(cxl-ui): cxl-marketing-nav menu bar button alignment on desktop

anoblet opened the pull request:
https://app.clickup.com/t/86b2hv84w

Desktop:
![localhost_6006_iframe html_args= id=cxl-ui-cxl-marketing-nav--cxl-marketing-nav viewMode=story](https://github.com/user-attachments/assets/146d52df-fa76-47b9-b635-86a5c96056a9)

Mobile:
![localhost_6006_iframe html_args= id=cxl-ui-cxl-marketing-nav--cxl-marketing-nav viewMode=story (1)](https://github.com/user-attachments/assets/fcb06189-63cd-40e8-a31a-bc18d93df63c)

pawelkmpt reviewed (changes requested):
It doesn't look right on desktop.

![Screenshot 2024-12-17 at 10 02 02](https://github.com/user-attachments/assets/c5b630af-5da1-4ad5-917d-f13ee19cbafb)
![Screenshot 2024-12-17 at 10 10 44](https://github.com/user-attachments/assets/b055f8a1-2625-469b-ad03-fedde4c243c9)
![Screenshot 2024-12-17 at 10 11 24](https://github.com/user-attachments/assets/98da8a25-e8c0-45f7-b9d5-e84bba1b7216)

anoblet commented:
I removed the faded red background for the close icon and changed the nav bar padding from `--lumo-space-m` to `--cxl-wrap-padding`. It's hard to get perfect alignment since the logo has whitespace in itself, though it looks better than before. The only issue I see is on desktop. It doesn't look possible to remove the padding or margin for the last menu item(https://stackoverflow.com/a/66664161/3935891) so the help link isn't aligned.

Mobile closed:
![localhost_](https://github.com/user-attachments/assets/3694ea92-0753-46a0-ab8f-35c224f479e6)

Mobile opened:
![localhost_ (1)](https://github.com/user-attachments/assets/22691116-2a80-424e-8dd1-18fcf75919e3)

Desktop:
![localhost_ (2)](https://github.com/user-attachments/assets/9709178b-ed73-48bb-ab6b-1710049a07fb)

pawelkmpt reviewed (commented):
@anoblet I deployed it live, looks good in most places but on the dashboard menu is wider than content. I see it uses `1c-w` layout.

Please check more non-standard pages on institute so we know if this is just dashboard issue or wider problem.

If just dashboard, propose solution

![Screenshot 2025-01-02 at 10 23 24](https://github.com/user-attachments/assets/f7aa70db-f83c-49ee-9fc6-53659bf0a4f5)

lkraav commented:
> It's hard to get perfect alignment since the logo has whitespace in itself, though it looks better than before.

Logo icon itself should absolutely have 0 / zero padding. Anything else is a bug.

anoblet commented:
It's difficult to test this locally since WP Starter has a different dashboard/layout, though the result should be this:

![cxl com_institute_dashboard_ (6)](https://github.com/user-attachments/assets/677b7b6a-f6c2-47f2-9e5e-a12cee88e089)

I set a max-width for the marketing nav equal to the max-width of the content.

> Logo icon itself should absolutely have 0 / zero padding. Anything else is a bug.

There's no left padding on the icon, though the SVG itself has whitespace:

<img width="51" alt="image" src="https://github.com/user-attachments/assets/0c97657a-5413-4efc-854b-b0937363acf7" />

pawelkmpt reviewed (changes requested):
Nav bottom border is not full width anymore. It's also visible on your screenshot @anoblet 

![Screenshot 2025-01-10 at 16 04 40](https://github.com/user-attachments/assets/b29e6cba-344f-484a-9a5f-5fc8b93b6c7e)

anoblet commented:
> Nav bottom border is not full width anymore. It's also visible on your screenshot @anoblet

This should be fixed. I moved the border to the marketing nav instead of the inner nav.

pawelkmpt reviewed (changes requested):
It's still broken. Now bottom border is not visible when the page loads. It appears when you start scrolling.

BEFORE
<img width="1508" alt="Screenshot 2025-01-14 at 17 35 07" src="https://github.com/user-attachments/assets/63676c80-d05d-4227-b300-8ab5a6792617" />

AFTER
<img width="1509" alt="Screenshot 2025-01-14 at 17 34 20" src="https://github.com/user-attachments/assets/f3327dd8-91ca-4abf-957f-ece0f1e869ce" />

anoblet commented:
I missed the `display: block` on `cxl-marketing-nav`. It should work now.

![localhost_ (3)](https://github.com/user-attachments/assets/d8ec6f8b-a42c-4f4e-9b42-62b4c54b67a7)
